src/serverMain/kotlin/com/studo/campusqr/endpoints/ReportData.kt

```diff
@@ -95,4 +93,38 @@ suspend fun AuthenticatedApplicationCall.returnReportData() {
           impactedUsersEmailsCsvFileName = "${csvFilePrefix?.plus("-emails") ?: "emails"}.csv"
       )
   )
+}
+
+suspend fun AuthenticatedApplicationCall.listAllActiveCheckIns() {
```

Comment:
Please add a description why we need this endpoint/function. Currently it looks like dead code that can be removed.